### PR TITLE
Fix dependency hell.

### LINKED ISSUE #602: Dependency hell
I was try ```composer install``` on both master and 2.3 and came to dependency hell.
From master:
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package liip/imagine-bundle dev-master exists as liip/imagine-bundle[0.x-dev, 1.0.0, 1.0.0-alpha1, 1.0.0-alpha2, 1.0.0-alpha3, 1.0.0-alpha4, 1.0.0-alpha5, 1.0.0-alpha6, 1.0.0-alpha7, 1.0.1, 1.0.2, 1.0.3, 1.0.4, 1.0.5, 1.0.6, 1.0.7, 1.0.8, 1.0.x-dev, 1.1.0, 1.1.1, 1.2.0, 1.2.1, 1.2.2, 1.2.3, 1.2.4, 1.2.5, 1.2.6, 1.2.7, 1.3.0, 1.3.1, 1.3.2, 1.3.3, 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.5.0, 1.5.1, 1.5.2, 1.5.3, 1.6.0, 1.7.0, 1.7.1, 1.7.2, 1.7.3, 1.7.4, 1.8.0, 2.0.x-dev, v0.10.0, v0.10.1, v0.11.0, v0.11.1, v0.12.0, v0.13.0, v0.14.0, v0.15.0, v0.15.1, v0.16.0, v0.17.0, v0.17.1, v0.18.0, v0.19.0, v0.20.0, v0.20.1, v0.20.2, v0.21.0, v0.21.1, v0.9.0, v0.9.1, v0.9.2, v0.9.3, v0.9.4] but these are rejected by your constraint.
  Problem 2
    - sonata-project/media-bundle 3.x-dev requires kriswallsmith/buzz ^0.15 -> satisfiable by kriswallsmith/buzz[v0.15] but these conflict with your requirements or minimum-stability.
    - sonata-project/media-bundle 3.5.1 requires knplabs/gaufrette ^0.1.6 || ^0.2 || ^0.3 -> satisfiable by knplabs/gaufrette[0.2.0, 0.2.1, v0.1.6, v0.1.7, v0.1.8, v0.1.9, v0.3, v0.3.1] but these conflict with your requirements or minimum-stability.
    - sonata-project/media-bundle 3.5.0 requires knplabs/gaufrette ^0.1.6 || ^0.2 -> satisfiable by knplabs/gaufrette[0.2.0, 0.2.1, v0.1.6, v0.1.7, v0.1.8, v0.1.9] but these conflict with your requirements or minimum-stability.
    - sonata-project/media-bundle 3.4.0 requires knplabs/gaufrette ^0.1.6 || ^0.2 -> satisfiable by knplabs/gaufrette[0.2.0, 0.2.1, v0.1.6, v0.1.7, v0.1.8, v0.1.9] but these conflict with your requirements or minimum-stability.
    - sonata-project/media-bundle 3.3.1 requires knplabs/gaufrette ^0.1.6 || ^0.2 -> satisfiable by knplabs/gaufrette[0.2.0, 0.2.1, v0.1.6, v0.1.7, v0.1.8, v0.1.9] but these conflict with your requirements or minimum-stability.
    - sonata-project/media-bundle 3.3.0 requires knplabs/gaufrette ^0.1.6 || ^0.2 -> satisfiable by knplabs/gaufrette[0.2.0, 0.2.1, v0.1.6, v0.1.7, v0.1.8, v0.1.9] but these conflict with your requirements or minimum-stability.
    - sonata-project/media-bundle 3.2.0 requires knplabs/gaufrette ^0.1.6 || ^0.2 -> satisfiable by knplabs/gaufrette[0.2.0, 0.2.1, v0.1.6, v0.1.7, v0.1.8, v0.1.9] but these conflict with your requirements or minimum-stability.
    - sonata-project/media-bundle 3.1.0 requires knplabs/gaufrette ^0.1.6 -> satisfiable by knplabs/gaufrette[v0.1.6, v0.1.7, v0.1.8, v0.1.9] but these conflict with your requirements or minimum-stability.
    - sonata-project/media-bundle 3.0.0 requires knplabs/gaufrette ^0.1.6 -> satisfiable by knplabs/gaufrette[v0.1.6, v0.1.7, v0.1.8, v0.1.9] but these conflict with your requirements or minimum-stability.
    - sonata-project/news-bundle dev-master requires sonata-project/media-bundle ^3.0 -> satisfiable by sonata-project/media-bundle[3.0.0, 3.1.0, 3.2.0, 3.3.0, 3.3.1, 3.4.0, 3.5.0, 3.5.1, 3.x-dev].
    - Installation request for sonata-project/news-bundle dev-master -> satisfiable by sonata-project/news-bundle[dev-master].
```

### PR DESCRIPTION
**Pull request by pcdummy**
Fixes #602 for 2.4-develop

Signed-off-by: René Jochum <rene@jochums.at>

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -26,10 +26,9 @@
         "knplabs/gaufrette": "~0.1.6",
         "knplabs/knp-menu-bundle": "dev-master",
         "mopa/bootstrap-bundle": "v3.0.0-beta3",
-        "nelmio/api-doc-bundle": "dev-master",
         "sensio/distribution-bundle": "~4.0",
-        "sensio/framework-extra-bundle": "^3.0.2",
-        "simplethings/entity-audit-bundle": "~0.5",
+        "sensio/framework-extra-bundle": "3.0.2",
```
Comment (OskarStark) [inline composer.json]: can you explain why you did this?

Comment (pcdummy) [reply composer.json]: Cause versions **v3.0.29** fails with:

      [RuntimeException]                                                                                
      You must enable the SensioFrameworkExtraBundle view annotations to use the ViewResponseListener.

Comment (pcdummy) [reply composer.json]: @OskarStark want me to find a fix for this?

Comment (OskarStark) [reply composer.json]: hmm IMO we should fix the deps in general, there are too many `@dev` dependencies 😢

**Comment by jordisala1991** [comment]
We merged recently a PR for the 3.0 branch, so this is no longer necessary.

**Pull request closed without merging**